init-scripts/faf-celery-beat.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=FAF Celery beat
+After=network.target
+
+[Service]
+Type=forking
+User=faf
+Group=faf
+EnvironmentFile=/etc/faf/celery-beat-env.conf
+WorkingDirectory=/etc/faf
+ExecStart=/usr/bin/python -m celery beat \
+    -A $CELERY_APP --pidfile=${CELERYD_PID_FILE} \
+    --logfile=${CELERYD_LOG_FILE} --loglevel=${CELERYD_LOG_LEVEL} \
+    --detach $CELERYD_OPTS
```

Comment:
Only works for me without --detach and without Type=forking (both are redundant)